src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -430,52 +434,72 @@ public void Initialize()
 
                 // take a snapshot so we can detect function additions/removals
                 _directorySnapshot = Directory.EnumerateDirectories(ScriptConfig.RootScriptPath).ToImmutableArray();
+                TraceWriter.Verbose("Script root directory snapshot captured.");
```

Comment:
Rather than have a separate log for this rather detailed operation, can we instead move this into your latency block below and view it as part of the "read functions metadata" operation which it is?

Reply:
Yeah I agree, will do.

Reply:
This is addressed in the latest PR